apps/emqx_conf/src/emqx_conf_schema.erl

```diff
@@ -1226,6 +1227,9 @@ collector_enabled(disabled, _) -> [].
 tr_gen_rpc_default_client_driver(Conf) ->
     conf_get("rpc.protocol", Conf).
 
+tr_gen_rpc_ssl_client_port(Conf) ->
+    conf_get("rpc.ssl_server_port", Conf).
```

Comment:
this is wrong, because in case one configure to use `manual`, and change server port for node2, then this config on node1 should be node2's server port, but not server port of itself.

Reply:
This is actually what this PR tries to solve, without this mapping, gen_rpc use the default port 5370. In regular manual mode, the server is listening on 5369.

https://github.com/emqx/gen_rpc/blob/46dd22100391f3d87daca989cde7e2ecc52a17b5/src/gen_rpc.app.src#L35

My theory is for intra-cluster communication each node 's gen_rpc client should connect to the same port that the server is listening on.  The port specified here is a *base* port, whatever the port discovery method in use , they should get aligned. 

In case of manual mode and all nodes have rpc.ssl_server_port set to `X` emqx1 listens on `X` emqx2 listens on `X` and the client of emqx2 connects to emqx1 should connect to`X`. 

In case of stateless mode and rpc.ssl_server_port set to `X` emqx1 listens on `X+1` emqx2 listens on `X+2` and the client of emqx2 connects to emqx1 should connect to`X+1`.  the client of emqx1 connected to emqx2 should connect to `X+2`


> this is wrong, because in case one configure to use manual, and change server port for node2, then this config on node1 should be node2's server port, but not server port of itself.

do you mean node1 rpc.ssl_server_port set to `X` and node2 rpc.ssl_server_port set to `Y` where X=/=Y?
this to me is a config error.

Reply:
right, it makes no sense to configure different ports when `manual`.
basically means no way to support more than one server port when there are more than 2 nodes.